src/studio-header/HeaderBody.tsx

```diff
@@ -135,6 +135,7 @@ const HeaderBody = ({
               logoutUrl,
               authenticatedUserAvatar,
               isAdmin,
+              isMobile,
```

Comment:
This is the only change that could have a runtime effect. Everything else is just test code / type checking.

But TypeScript correctly pointed out that we weren't passing the required `isMobile` here. I assume it's correct ot just pass it in, but ignoring the real value and passing `isMobile: false` would be more consistent with how the code worked before.